bitnami/redis-cluster/templates/NOTES.txt

```diff
@@ -0,0 +1,121 @@
+** Please be patient while the chart is being deployed **
+
+{{- if contains .Values.service.type "LoadBalancer" }}
+{{- if not .Values.usePassword }}
+{{ if and (not .Values.networkPolicy.enabled) (.Values.networkPolicy.allowExternal) }}
+
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "service.type=LoadBalancer" and "usePassword=false" you have
+    most likely exposed the Redis service externally without any authentication
+    mechanism.
+
+    For security reasons, we strongly suggest that you switch to "ClusterIP" or
+    "NodePort". As alternative, you can also switch to "usePassword=true"
+    providing a valid password on "password" parameter.
+
+-------------------------------------------------------------------------------
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{ if .Values.usePassword }}
+To get your password run:
+
+    export REDIS_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "redis.secretName" . }} -o jsonpath="{.data.redis-password}" | base64 --decode)
+{{- end }}
+
+{{- if and .Values.cluster.enabled .Values.cluster.externalAccess.enabled }}
```

Comment:
I think that in this chart it does not make sense to have the standalone option. That's what the regular redis chart is for. I would remove the cluster.enabled section and make it a cluster by default

Reply:
I think that right now, before continuing, I would remove the standalone options so the chart becomes simpler

Reply:
Done